tests/testthat/test-verified-ttestindependentsamples.R

```diff
@@ -31,31 +31,32 @@ test_that("Main table results match R, SPSS, SAS and Minitab 1", {
 })
 
 # https://jasp-stats.github.io/jasp-verification-project/t-tests.html#independent-samples-t-test
-test_that("Descriptives table results match R, SPSS, SAS and Minitab 1", { 
+test_that("Descriptives table results match R, SPSS, SAS and Minitab 1", {
   # Descriptives
   resultTable <- results$results$ttestDescriptives$collection$ttestDescriptives_table$data
-  
+
   jaspTools::expect_equal_tables(
-    "test"=resultTable, 
-    "ref"=list("TRUE", 125, 1, 1.8, 0.7, 0.0626099033699941, "Score", "FALSE",
-               150, 2, 1.6, 0.9, 0.0734846922834953, "Score"))
+    "test"=resultTable,
```

Comment:
Technically we should be verifying the values in the verified test files.

@AlexanderLyNL what do we do about this? We need to update this entry now to merge this PR, but then the test will contain a value that has not been verified by @JTPetter.